packages/static-website/src/cloudfront-web-acl.ts

```diff
@@ -118,7 +118,7 @@ export class CloudfrontWebAcl extends Construct {
   private createOnEventHandler(stack: Stack, aclName: string): Function {
     const onEventHandlerName = `${PDKNag.getStackPrefix(stack)
       .split("/")
-      .join("-")}OnEventHandler`;
+      .join("-")}OnEventHandler-${this.node.addr.slice(0, 4)}`;
```

Comment:
Looks like it was explicit so that it could be used for a more restrictive policy for writing to cloudwatch logs. I don't think there's any harm in moving the wildcard to the `/aws/lambda/*` log prefix though, so will update to a dynamic name :)